opentreemap/importer/tasks.py

```diff
@@ -86,13 +86,13 @@ def run_import_event_validation(import_type, import_event_id, file_obj):
         return
 
     ie.status = GenericImportEvent.PREPARING_VERIFICATION
-    ie.save()
+    ie.update_progress_timestamp_and_save()
 
     try:
-        row_set = ie.rows()
-        validation_tasks = (
-            _validate_rows.subtask(row_set[i:(i+settings.IMPORT_BATCH_SIZE)])
-            for i in xrange(0, ie.row_count, settings.IMPORT_BATCH_SIZE))
+        validation_tasks = []
+        for i in xrange(0, ie.row_count, settings.IMPORT_BATCH_SIZE):
+            args = (import_type, ie.id, i)
+            validation_tasks.append(_validate_rows.subtask(args))
```

Comment:
I tried it this way and it did not work. I think it has something to do with the way celery handles arg passing to subtasks.